src/Materials/shaderMaterial.ts

```diff
@@ -862,6 +864,40 @@ export class ShaderMaterial extends Material {
         return false;
     }
 
+    private BJSNODEMATERIALEDITOR = NodeMaterial._getGlobalNodeMaterialEditor();

```

Comment:
I'm REALLY not a big fan on introducing this dependency :(
If I'm playing with treeshacking, I do not want to get all the NME when only shadermaterial is required